.github/workflows/release.yml

```diff
@@ -54,6 +55,11 @@ jobs:
     needs: [e2e-tests]
     steps:
       - uses: actions/checkout@v4
+      - name: Setup certificate
```

Comment:
Everything seems to work, however the release.yml is not a valid file. 
![Captura de pantalla 2024-04-17 204530](https://github.com/Arquisoft/wiq_en3b/assets/124174254/f0ff0f1a-b83b-494d-a090-737e481b1869)

Reply:
I see, it's a problem with the indentation, thanks for noticing!